vanilla_first_setup/window.py

```diff
@@ -55,13 +58,30 @@ def __init__(self, post_script: str, user: str, new_user: bool = False, **kwargs
         # True/False = managed result
         self.__last_result = None
 
+        # Create and run the tests to make sure the previous run installed
+        # all the packages properly
+        if self.__init_mode == 1:
+            self.__tests = Tests()
+            self.__tests.load()
+
+            # 2 = succeeded
+            # 1 = did not succeed
+            # 0 = did not test
+            self.__tests_succeeded = 2 if self.__tests.test() else 1
+            post_script = 0 if self.__tests_succeeded == 1 else 1
```

Comment:
More of a stylistic thing, but I would change the numbers to match exit codes, so 0=succeeded, 1=failed, -1=did not test